App.tsx

```diff
@@ -1,14 +1,24 @@
-import { StatusBar } from 'expo-status-bar';
-import { Text, View } from 'react-native';
+import { StatusBar } from 'expo-status-bar'
+import { Text, View } from 'react-native'
+import { LinearGradient } from 'expo-linear-gradient'
 
 // custom imports
-import { styles } from './baseStyle';
+import { styles } from './baseStyle'
 
-export default function App() {
+export default function App () {
   return (
-    <View style={styles.container}>
-      <Text>Open up App.tsx to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
-  );
+    <LinearGradient
```

Comment:
is it possible to put the linear gradient style definition in our baseStyle.tsx? if it's not possible that's also alright

Reply:
@oyiza which style definition, can you point out the line of code?

Reply:
actually nevermind. i was referring to the entire linearGradient component but the style is already in the styles file

Reply:
yup!